README.md

```diff
@@ -1,4 +1,7 @@
 React Router Docs
+
+[![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors)
```

Comment:
`all-contributors-cli` will update the number here automagically when you run `npm run update-contributors`